ibridgesgui/__main__.py

```diff
@@ -302,5 +302,10 @@ def main():
     app.exec()
 
 
-if __name__ == "__main__":
+def main_deprecated():
+    """Deprecate the ibridges-gui call."""
+    print("'ibridges-gui' will be deprecated. Please use 'ibridges gui' from ibridges 1.5.1")
     main()
+
+if __name__ == "__main__":
```

Comment:
This should call the normal main right?

Reply:
Then the main needs to call the deprecation_main ... Is that what you mean?

Reply:
The part with `if __name__ == "__main__"` should call the non-deprecated version, not `deprecated_main`.